packages/rendering/server.ts

```diff
@@ -39,10 +41,7 @@ const renderArticle = ({ body }: express.Request, res: express.Response) => {
 
 const renderAMPArticle = ({ body }: express.Request, res: express.Response) => {
     try {
-        const resp = AMPDocument({
-            bodyHTML: '<h1>This is a valid AMP doc!</h1>',
-        });
-
+        const resp = AMPDocument({ body: React.createElement(AMPArticle) });
```

Comment:
FWIW I think this is a nicer approach than hardcoding the page in the document.

Reply:
Yeah, only ugliness was having to call React.createElement but doesn't seem too bad.

Reply:
We could rename this `server.tsx` to give us JSX support?

Reply:
I'm not really a fan of us calling react.createElement here. Can we have a seperate tsx file which composes the template and component into our response?

Reply:
Okay gone with tsx'ing the file for now.